Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -322,6 +322,21 @@ TEST_F(SerializerTest, BadStringValue) {
       Status(FirestoreErrorCode::InvalidArgument, "ignored"), bytes);
 }
 
+TEST_F(SerializerTest, BadStringValue2) {
+  std::vector<uint8_t> bytes;
+  Status status =
+      serializer.EncodeFieldValue(FieldValue::StringValue("a"), &bytes);
+  EXPECT_TRUE(status.ok());
+
+  // Mutate that bytes: Claim that the string length is 5 instead of 1.
```

Comment:
Typo in "that". Probably "the"?

Consider dropping "Mutate (that|the) bytes: " anyway--that's pretty obviously what the next statement is doing and the rest of your comment is describing why.